ncm-nss/src/main/perl/nss.pm

```diff
@@ -95,16 +95,16 @@ sub Configure {
 
     my @new = ( "# Generated by ncm-nss\n\n" );
     push(@new, map { "$_: $databases->{$_}\n" } sort keys %$databases);
-    my $contents = join("", @new);
-    # cannot use CAF::FileWriter because close() always returns false in
-    # noaction mode so cannot show when nscd would be restarted
-    my $result = LC::Check::file("/etc/nsswitch.conf",
-                                 backup => ".OLD",
-                                 contents => $contents,
-                                 owner => "root",
-                                 group => "root",
-                                 mode => "0444",
-                                );
+    my $fh = CAF::FileWriter->open("/etc/nsswitch.conf",
+                                   backup => ".OLD",
+                                   owner => "root",
+                                   group => "root",
+                                   mode => "0444",
```

Comment:
Thanks for checking. Something is weird with maven on this one machine. Using another also works. :(